pages/dkp/kommander/2.2/install/air-gapped/catalog/index.md

```diff
@@ -239,8 +241,8 @@ It may take a while to push all the images to your image registry, depending on
     ```bash
     kommander install --installer-config ./install.yaml \
```

Comment:
maybe clean up the old kommander commander to use the dkp command here as well?

Reply:
I think that's being done in https://github.com/mesosphere/dcos-docs-site/pull/4232 🤔

Reply:
to avoid conflicts, I'll defer this to the other PRs being worked on to change these